server/channels/app/desktop_login.go

```diff
@@ -14,8 +14,9 @@ func (a *App) GenerateAndSaveDesktopToken(createAt int64, user *model.User) (*st
 	err := a.Srv().Store().DesktopTokens().Insert(token, createAt, user.Id)
 	if err != nil {
 		// Delete any other related tokens if there's an error
-		a.Srv().Store().DesktopTokens().DeleteByUserId(user.Id)
-
+		if deleteErr := a.Srv().Store().DesktopTokens().DeleteByUserId(user.Id); deleteErr != nil {
+			return nil, model.NewAppError("GenerateAndSaveDesktopToken", "app.desktop_token.delete.error", nil, "", http.StatusInternalServerError).Wrap(deleteErr)
+		}
```

Comment:
By returning an error here, we obfuscate the error returned by `a.Srv().Store().DesktopTokens().Insert`. I think we should log the error instead. 

What do you think @devinbinnie ?

Reply:
Sorry I must have misread this, yeah we should really be returning the `Insert` error here instead, as that's the real issue

Reply:
I will fix it ASAP